go-controller/pkg/types/const.go

```diff
@@ -171,6 +171,10 @@ const (
 	TopologyExternalID = OvnK8sPrefix + "/" + "topology"
 	// key for topology version external-id
 	TopologyVersionExternalID = "k8s-ovn-topo-version"
+	// key for load_balancer kind external-id
+	LoadBalancerKindExternalID = "k8s.ovn.org/kind"
+	// key for load_balancer service external-id
+	LoadBalancerOwnerExternalID = "k8s.ovn.org/owner"
```

Comment:
there is one more place where it is actually set https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/util/kube.go#L404-L405
and we may want to redefine this as `types.OvnK8sPrefix + "/owner"` .

Reply:
whoa, I missed that. I will fix it; otherwise it will feel like a splinter, ;)

Reply:
done.